app/src/main/java/fr/neamar/kiss/adapter/RecordAdapter.java

```diff
@@ -61,13 +68,61 @@ public View getView(int position, View convertView, ViewGroup parent) {
         return results.get(position).display(getContext(), results.size() - position, convertView);
     }
 
-    public void onLongClick(int pos) {
+    public void onLongClick(final int pos, View v) {
+        // Check that the pojo is an app
+        if (getItemViewType(pos) == 0)
```

Comment:
hum, i'd rather use inheritance.

Maybe add a `getPopup()` on Result, override the method in AppResult to return your popup?
That way you can keep `appPojo` private and get distinct menu per record, without coupling tightly this class with appRecord